testsuite/shared/src/main/java/org/wildfly/test/jmx/ServiceActivatorDeployment.java

```diff
@@ -127,4 +116,74 @@ public void stop(StopContext context) {
     public Void getValue() throws IllegalStateException, IllegalArgumentException {
         return null;
     }
+
+    private ObjectInstance registerMBean(Object mbean, ObjectName name) throws Exception {
+        return inVmActions().registerMBean(mbeanServerValue.getValue(), mbean, name);
```

Comment:
Note to self: Darran and I chatted about how this is a problem. It's nice if people want it for some situations, but requiring new steps to use JMX, particular outside the jboss.ass[.expr] domains, is a problem.